uwp/Smith-Chart/Series.md

```diff
@@ -144,3 +144,29 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![SfSmithChart Series Visibility](Series_images/Series_img3.png)
+
+## Data plotting customization
+
+By default, the data points are plotted by sorting them based on the resistance values and it can be plotted based on their index value as it is by setting the [`ArrangeByIndex`]() property as true.
```

Comment:
By default, the data points are plotted by sorting them based on the resistance values and they can be plotted based on their index values as it is by setting the [`ArrangeByIndex`]() property as true.

Reply:
I have updated.